pkg/models/actions/services/ark_sechub_exec_action_consts.go

```diff
@@ -0,0 +1,125 @@
+package services
+
+import (
+	"github.com/cyberark/ark-sdk-golang/pkg/models/actions"
+	sechubconfiguration "github.com/cyberark/ark-sdk-golang/pkg/models/services/sechub/configuration"
+	sechubfilters "github.com/cyberark/ark-sdk-golang/pkg/models/services/sechub/filters"
+	sechubscans "github.com/cyberark/ark-sdk-golang/pkg/models/services/sechub/scans"
+	sechubsecrets "github.com/cyberark/ark-sdk-golang/pkg/models/services/sechub/secrets"
+	sechubsecretstores "github.com/cyberark/ark-sdk-golang/pkg/models/services/sechub/secretstores"
+	sechubsyncpolicies "github.com/cyberark/ark-sdk-golang/pkg/models/services/sechub/syncpolicies"
+)
+
+// ServiceInfoActionToSchemaMap is a map that defines the mapping between Sec Hub Service Info action names and their corresponding schema types.
+var ServiceInfoActionToSchemaMap = map[string]interface{}{
+	"get-service-info": nil,
```

Comment:
Usually we do not use "get" keyword, so all getters should just be "service-info" etc